src/modules/CreateArticle.js

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+
+const CreateArticle = {
```

Comment:
I'm NOT on board with this naming. I would not use the word `Create` here. You have it as a function name below, so I would keep it more generic.

Reply:
Changed to `ArticlesServices`